app/views/home/_search.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="dgu-split-background__top">
+  <div class="dgu-split-background__top-inner">
+    <div class="dgu-home-search">
+      <div class="grid-row">
+        <div class="column-two-thirds">
+          <h1 class="heading-xlarge">
+            <%= t('.find_gov_data') %>
+          </h1>
+          <p class="lede">
+            <%= t('.lede') %>
+          </p>
+        </div>
+      </div>
+      <div class="grid-row">
+        <div class="column-two-thirds dgu-search-box">
+          <form action="/search" method="GET">
+            <label for="q" class="visuallyhidden"><%= t('.accessibility.search_box_label') %></label>
+            <!-- Make sure there is no space between the input and button else the button will become detached --> 
```

Comment:
I would put the comment where there's not suppose to be space:
`<input/><!-- don't add any whitespace here --><button>